conf/wdio-docker-parallel.conf.js

```diff
@@ -0,0 +1,28 @@
+var defaults = require("./wdio.conf.js");
+var _ = require("lodash");
+
+var overrides = {
+  specs: [
+    './test/specs/plain/**/*.js'
```

Comment:
keep the path generic in all the conf files.
Either it should be "./test/specs/plain/**/.js" or "./test/specs/plain/.js"